melusine/models/train.py

```diff
@@ -549,7 +592,7 @@ def _prepare_data(self, X, y, validation_data=None):
         return X_input, y_categorical
 
     def _get_meta(self, X):
```

Comment:
the type of _get_meta return as pd.DataFrame caused a bug with TFP.
It can be a numpy array instead (no problem with base melusine/keras)